.github/workflows/mac_release.yml

```diff
@@ -153,39 +267,96 @@ jobs:
         include:
           - macos_dev_target: 12.1
             os: macos-12
+            arch: x86_64
             python-arch: x64
             test_key: mac12
           - macos_dev_target: 13.0
             os: macos-14
+            arch: arm64
             python-arch: arm64
             test_key: mac13-arm64
 
     steps:
-      - uses: actions/checkout@v4  # Still need E+ checked out to get testing scripts
-        with:
-          path: checkout
-
-      - name: Set up Python ${{ env.Python_REQUIRED_VERSION }}
-        uses: actions/setup-python@v5
-        id: setup-python
-        with:
-          python-version: ${{ env.Python_REQUIRED_VERSION }}
-          architecture: ${{ matrix.python-arch }}
-
-      - name: Gather Test Package from Artifacts
-        uses: actions/download-artifact@v4
-        with:
-          name: energyplus-${{ matrix.os }}
-          path: package
-
-      - name: Check Contents
-        shell: bash
-        run: ls
-
-      - name: Check Package contents
-        shell: bash
-        working-directory: package
-        run: ls
-
-      - name: Run Package Tests
-        run: python checkout/scripts/package_tests/runner.py --verbose ${{ matrix.test_key }} package/
+    - uses: actions/checkout@v4  # Still need E+ checked out to get testing scripts
```

Comment:
This is just whitespace change, use https://github.com/NREL/EnergyPlus/pull/10645/files?w=1 to not see them.